diff --git a/bash_aliases.sh b/bash_aliases.sh
index b7c5daf..4239a41 100644
--- a/bash_aliases.sh
+++ b/bash_aliases.sh
@@ -54,6 +54,7 @@ alias m='less'
 #alias mct='mvn verify -Pcomponent-tests -P \!update-tisp-dependencies,\!update-parent'
 alias mci='mvn clean install -P \!update-tisp-dependencies,\!update-parent'
 alias mct='mvn clean install -Pcomponent-tests,\!update-tisp-dependencies,\!update-parent'
+alias mrun='mvn spring-boot:run'
 alias mtf='mvn -P component-tests-fast verify'
 alias mroe='less'
 alias p3='/usr/local/bin/python3'
